routes/markers.js

```diff
@@ -308,6 +311,7 @@ module.exports = function(connection) {
                     else {
                         res.setHeader('Content-Type', 'application/json');
                         res.json(data);
+                        utils.addRSS(connection, 'Marker Deleted', `OpenMRS Atlas bids farewell to ${data.name}`, req.session.user.uid);
```

Comment:
* Title: OpenMRS Atlas bids farewell to ${data.name}
* Description: ${username} removed ${data.name} from the OpenMRS Atlas.

The link for this one would, obviously, not point to the marker; rather, just be to https://atlas.openmrs.org/.